v2/contribute/decisions/thirdparty/0009-config-management-functions-when-recipe-is-disabled.mdx

```diff
@@ -0,0 +1,40 @@
+---
+id: "0009"
+title: Recipe config management functions when the recipe is disabled in core
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Recipe config management functions when the recipe is disabled in core
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-08" created="2022-12-08" deciders={["rishabhpoddar", "sattvikc"]} proposedBy={["sattvikc"]} />
+
+## Context and Problem Statement
+
+In the context of thirdparty, should management of dynamic provider list be allowed when the thirdparty recipe is disabled in core?
+
+## Considered Options
+
+* Should not be allowed
+* Should be allowed
+
+
+## Decision Outcome
+
+Chosen option: Should be allowed, because
+
+* it is more flexible
+
+## Pros and Cons of the Options
+
+### Should not be allowed
+
+<ArgumentCon> Ambiguity when the user enables a recipe, adds the config and then disables it. Should already configured values be retained or destroyed? </ArgumentCon>
```

Comment:
they don't have to be destroyed per say.. just not be returned. This would have the same effect as what we have now - just with a worse developer experience.